mlflow/server/js/src/components/modals/RenameRunFormView.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+import { withRouter } from 'react-router-dom';
+import { Formik, Field } from 'formik';
+
+import { validationSchema } from './validation';
+
+/**
+ * Component that renders a form for updating a run's name. Expects to be 'closeable'
+ * (i.e. rendered within a closeable dialog) and so accepts an `onClose` callback.
+ */
+class RenameRunFormView extends Component {
+  static propTypes = {
+    onSubmit: PropTypes.func.isRequired,
+    onClose: PropTypes.func.isRequired,
+    runName: PropTypes.string.isRequired,
+    experimentId: PropTypes.number.isRequired
+  }
+
+  renderForm = (renderProps) => {
+    const {
+      handleSubmit,
+      isSubmitting,
+    } = renderProps;
+    const saveText = isSubmitting ? "Saving..." : "Save";
+    return <form onSubmit={handleSubmit} style={styles.form}>
+      <div style={styles.formField}><h2 style={{"marginTop": "0px"}}>Rename Run</h2></div>
```

Comment:
This is basically the modal header? It's a bit brittle to reuse styles.formField here. I wonder if it's possible to use bootstrap styles here.